### PR TITLE
pr split 10 - add 10 files

### PR DESCRIPTION
**Pull request by michaelkernaghan**
Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have added unit tests
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: integration-tests/rpc-get-protocol-constants.spec.ts [hunk 1]
```diff
@@ -508,43 +510,19 @@ CONFIGS().forEach(({ lib, protocol, rpc }) => {
...
-                tx_rollup_rejection_max_proof_size: 30000,
-                tx_rollup_sunset_level: 3473409,
-                tx_rollup_withdraw_period: 40000,
+                delay_increment_per_round: new BigNumber(5)
             });
 
             done();
         })
 
-        kathmandunet(`successfully fetches all constants for kathmandunet using ${rpc}`, async (done) => {
```
Comment (roxaneletourneau) [inline integration-tests/rpc-get-protocol-constants.spec.ts:541]: test for kathmandunet should not be removed

File: integration-tests/rpc-nodes.spec.ts [hunk 2]
```diff
@@ -31,12 +32,292 @@ CONFIGS().forEach(
...
+                done();
+            });
+
+            it(`Verify that rpcClient.getNormalizedScript for known contract returns an access the script of the contract and normalize it using the requested unparsing mode`, async (done) => {
+                const script = await rpcClient.getNormalizedScript(knownContract);
+                expect(script).toBeDefined();
+                done();
+            });
+
+            it(`Verify that pcClient.getContract returns an access the complete status of a contract`, async (done) => {
```
Comment (roxaneletourneau) [inline integration-tests/rpc-nodes.spec.ts:73]: pcClient -> rpcClient

File: integration-tests/rpc-nodes.spec.ts [hunk 3]
```diff
@@ -31,12 +32,292 @@ CONFIGS().forEach(
...
+                done();
+            });
+
+            it('Verify that rpcClient.getBlockMetadata returns all metadata associated to the block', async (done) => {
+                const blockMetadata = await rpcClient.getBlockMetadata();
+                expect(blockMetadata).toBeDefined();
+                done();
+            });
+
+            it('Verify that pcClient.getBakingRights retrieves the list of delegates allowed to bake a block', async (done) => {
```
Comment (roxaneletourneau) [inline integration-tests/rpc-nodes.spec.ts:151]: pcClient -> rpcClient

File: integration-tests/rpc-nodes.spec.ts [hunk 4]
```diff
@@ -31,12 +32,292 @@ CONFIGS().forEach(
...
+                    };
+
+                    await rpcClient.preapplyOperations([operation]);
+                } catch (ex: any) {
+                    expect(ex.message).toMatch('contract.counter_in_the_past');
+                }
+                done();
+            });
+
+            it('erify that rpcClient.getEntrypoints for known contract returns list of entrypoints of the contract', async (done) => {
```
Comment (roxaneletourneau) [inline integration-tests/rpc-nodes.spec.ts:271]: erify -> Verify

File: integration-tests/rpc-nodes.spec.ts [hunk 5]
```diff
@@ -31,12 +32,292 @@ CONFIGS().forEach(
...
+            });
+
+            it('getSuccessorPeriod', async (done) => {
+                const successorPeriod = await rpcClient.getSuccessorPeriod();
+                expect(successorPeriod).toBeDefined();
+                done();
+            });
         });
 
         it('List the ancestors of the head block', async (done) => {
```
Comment (roxaneletourneau) [inline integration-tests/rpc-nodes.spec.ts:323]: This test and the following ones seem to be duplicated.

File: integration-tests/rpc-taquito.spec.ts [hunk 6]
```diff
@@ -5,22 +5,23 @@ import { RpcClient } from '../packages/taquito-rpc/src/taquito-rpc';
...
-            it('should get a block using head and tilde syntax', async (done) => {
+            it('Verify that client.getBlock({ block: \'head~2\' }) returns a block using head and tilde syntax', async (done) => {
                 const block = await client.getBlock({ block: 'head~2' });
                 const blockHeader = await client.getBlockHeader({ block: block.hash })
 
                 expect(block.header.predecessor).toEqual(blockHeader.predecessor);
                 done();
             })
-            it('should get a block using hash and tilde syntax', async (done) => {
+            it('Verify that client client.getBlock({ block: `${block.hash}~1` }) returns a block using hash and tilde syntax', async (done) => {
```
Comment (roxaneletourneau) [inline integration-tests/rpc-taquito.spec.ts:24]: client ~client~

File: integration-tests/tzip16-metadata-view.spec.ts [hunk 7]
```diff
@@ -8,13 +8,13 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
 	Tezos.addExtension(new Tzip16Module());
 	const test = require('jest-retries');
```
Comment (roxaneletourneau) [inline integration-tests/tzip16-metadata-view.spec.ts:9]: Remove jest-retries?

**Review by roxaneletourneau — CHANGES_REQUESTED** [review]

File: integration-tests/rpc-get-protocol-constants.spec.ts (hunk 1 above)
Comment (michaelkernaghan) [reply integration-tests/rpc-get-protocol-constants.spec.ts:541]: i had to merge with master to get the kathmandunet stuff back into the branch

File: integration-tests/rpc-nodes.spec.ts (hunk 2 above)
Comment (michaelkernaghan) [reply integration-tests/rpc-nodes.spec.ts:73]: fixed

File: integration-tests/rpc-nodes.spec.ts (hunk 3 above)
Comment (michaelkernaghan) [reply integration-tests/rpc-nodes.spec.ts:151]: fixed

File: integration-tests/rpc-nodes.spec.ts (hunk 5 above)
Comment (michaelkernaghan) [reply integration-tests/rpc-nodes.spec.ts:323]: removed duplicate tets

File: integration-tests/rpc-taquito.spec.ts (hunk 6 above)
Comment (michaelkernaghan) [reply integration-tests/rpc-taquito.spec.ts:24]: fixed

File: integration-tests/rpc-get-protocol-constants.spec.ts [hunk 8]
```diff
@@ -592,6 +592,64 @@ CONFIGS().forEach(({ lib, protocol, rpc }) => {
 
     describe(`Fetch constants for testnet`, () => {
 
+        ithacanet(`successfully fetches all constants for ithacanet using ${rpc}`, async (done) => {
```
Comment (roxaneletourneau) [inline integration-tests/rpc-get-protocol-constants.spec.ts:595]: ithacanet not needed anymore

File: integration-tests/rpc-nodes.spec.ts (hunk 5 above)
Comment (roxaneletourneau) [reply integration-tests/rpc-nodes.spec.ts:319]: @michaelkernaghan It seems the duplicates are still there

File: integration-tests/rpc-nodes.spec.ts [hunk 9]
```diff
@@ -90,11 +365,10 @@ CONFIGS().forEach(
         });
 
         it('Executes tzip4 views by calling runView ', async (done) => {
-          
           let chainId: string;
 
-          if (protocol === Protocols.PtKathman) {
-            chainId = ChainIds.KATHMANDUNET
+          if (protocol === Protocols.Psithaca2) {
```
Comment (roxaneletourneau) [inline integration-tests/rpc-nodes.spec.ts:370]: if (protocol === Protocols.PtKathman) {
            chainId = ChainIds.KATHMANDUNET
          } else {
            chainId = ChainIds.JAKARTANET2
          }

**Review by roxaneletourneau — CHANGES_REQUESTED** [review]

File: integration-tests/rpc-get-protocol-constants.spec.ts (hunk 8 above)
Comment (michaelkernaghan) [reply integration-tests/rpc-get-protocol-constants.spec.ts:595]: removed ithacanet

File: integration-tests/rpc-nodes.spec.ts (hunk 9 above)
Comment (michaelkernaghan) [reply integration-tests/rpc-nodes.spec.ts:370]: changed as recommended

**Pull request merged**